test/unit/v1beta1/suggestion/test_hyperopt_service.py

```diff
@@ -176,6 +177,72 @@ def test_get_suggestion(self):
+                            parameter_type=api_pb2.DOUBLE,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="10", min="5", list=[], step="0.8", distribution=api_pb2.UNIFORM)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-8",
+                            parameter_type=api_pb2.DOUBLE,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="10", min="5", list=[], distribution=api_pb2.UNIFORM)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-9",
+                            parameter_type=api_pb2.DOUBLE,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="10", min="5", list=[], step="0.8", distribution=api_pb2.NORMAL)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-10",
+                            parameter_type=api_pb2.DOUBLE,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="10", min="5", list=[], step="0.8", distribution=api_pb2.LOG_NORMAL)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-11",
+                            parameter_type=api_pb2.INT,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="64", min="32", distribution=api_pb2.LOG_NORMAL)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-12",
+                            parameter_type=api_pb2.INT,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="5", min="1", list=[], distribution=api_pb2.UNIFORM)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-13",
+                            parameter_type=api_pb2.INT,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="10", min="5", list=[], step="0.8", distribution=api_pb2.UNIFORM)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-14",
+                            parameter_type=api_pb2.INT,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="10", min="5", list=[], distribution=api_pb2.LOG_UNIFORM)
+                        ),
+                        api_pb2.ParameterSpec(
+                            name="param-15",
+                            parameter_type=api_pb2.INT,
+                            feasible_space=api_pb2.FeasibleSpace(
+                                max="100", min="5", list=[], step="0.8", distribution=api_pb2.NORMAL)
```

Comment:
```suggestion
                                max="100", min="5", list=[], distribution=api_pb2.NORMAL)
```

We should verify the case where the step is not specified and distribution is Normal.